src/fri/fri_layer.cairo

```diff
@@ -36,19 +36,18 @@ fn compute_coset_elements(
     ref sibling_witness: Span<felt252>,
     coset_size: felt252,
     coset_start_index: felt252,
-    mut offset_within_coset: felt252,
```

Comment:
We should also update Inputs documentation above

Reply:
Good point